libs/SalesforceSDK/src/com/salesforce/androidsdk/security/PasscodeManager.java

```diff
@@ -462,34 +479,22 @@ public int getMinPasscodeLength() {
     /**
      * @param ctx
      * @param minPasscodeLength
-     * @return true if a passcode change is required and the app is entering a locked state.
      */
-    public boolean setMinPasscodeLength(Context ctx, int minPasscodeLength) {
-        boolean passcodeChangeRequired = false;
+    public void setMinPasscodeLength(Context ctx, int minPasscodeLength) {
     	if (minPasscodeLength > this.minPasscodeLength) {
-            this.minPasscodeLength = minPasscodeLength;
-
-            /*
-             * This needs to happen only if a passcode exists, in order to trigger
-             * the 'Change Passcode' flow. Otherwise, we simply need to update
-             * the minimum length in memory. The 'Create Passcode' flow is
-             * triggered later from OAuthWebviewHelper.
-             */
             if (hasStoredPasscode(ctx)) {
-                showLockActivity(ctx, true);
```

Comment:
The next SalesforceActivity that gets resumed will ask the passcode manager if the app needs to be locked - that's when the screen will be shown.